### PR TITLE
[MOB-15254] ACS Doc updates

### PR DESCRIPTION
**Pull request by rymorale**
Updates based on the doc guidelines

### CODE REVIEW

File: using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md [hunk 1]
```diff
@@ -107,15 +119,20 @@ Campaign.registerExtension();
 ```
 {% endtab %}
 
+{% tab title="iOS \(AEP 3.x\)" %}
```
Comment (calebk1) [inline using-mobile-extensions/adobe-campaign-standard/adobe-campaign-standard-api-reference.md:122]: We don't need to add the \\ to the title, as it's already escaped.

**Review by calebk1 — COMMENTED** [review]
Looks good to me! Just a few minor comments.

File: using-mobile-extensions/adobe-campaign-standard/README.md [hunk 2]
```diff
@@ -613,32 +810,31 @@ The website URL in the local notification response can be loaded using the [open
 }
 ```
 
-#### Swift
+#### Handling push notification website URLs on iOS
+
+### Swift
```
[the remark this answers is not in the document]
Comment (rymorale) [reply using-mobile-extensions/adobe-campaign-standard/README.md:815]: Ah didn't notice that. I'll make the headings H2 and the child headings H3.

**Comment by sbenedicadb** [comment]
@calebk1 looks like ryan has made the updates you requested.  can we merge this PR?  thanks.

**Review by calebk1 — APPROVED** [review]
LGTM!

**Pull request merged**